src/fetch/dune.py

```diff
@@ -128,12 +115,9 @@ def get_service_fee_status(self) -> list[DuneRecord]:
         """
         Fetches & Returns Parsed Results for VouchRegistry query.
         """
-        pool_values = ",\n".join(RECOGNIZED_BONDING_POOLS)
         return self._get_query_results(
             query=self._parameterized_query(
                 query_data=QUERIES["SERVICE_FEE_STATUS"],
-                params=[
-                    QueryParameter.text_type("BondingPoolData", pool_values),
-                ],
+                params=[],
```

Comment:
I would have expected not passing this parameter at all would also have worked. But I think you tested a few things already, right?

Reply:
Yeah i believe it was complaining